docs/changelog.rst

```diff
@@ -8,6 +8,9 @@ hipBLASLt
 hipBLASLtExt
 =============
 
+2023-08-10
+----------
```

Comment:
Changed feature follow by RCOM version but not by date

Reply:
Which version? 5.7?